content/influxdb/v2.0/visualize-data/visualization-types/band.md

```diff
@@ -114,31 +115,57 @@ To view **Band** controls, click **{{< icon "gear" >}} Customize** next to the v
   - **Smooth**: Display a time series in a line graph with smooth point interpolation.
   - **Step**: Display a time series in a staircase graph.
 - **Line Colors**: Select a color scheme to use for your graph.
 - **Hover Dimension**: Select the data to display in the tooltip when you hover over the graph:
-  - **auto** or **X Axis**: Show all points with the same x value along the y-axis.
-  - **Y Axis**: Show all points with the same y value along the x-axis.
-  - **X & Y Axis**: Show only the point being currently hovered over.
-
-###### Y Axis
-- **Y Axis Label**: Label for the y-axis.
-- **Y Value Unit Prefix**:
-  - **None**: No prefix.
-  - **SI**: International System of Units (SI) or metric prefix.
-  - **Binary**: Binary multiple prefix.
-- **Y Axis Prefix**: Prefix to be added to y-value.
-- **Y Axis Suffix**: Suffix to be added to y-value.
-- **Y Axis Domain**: The y-axis value range.
-  - **Auto**: Automatically determine the value range based on values in the data set.
-  - **Custom**: Manually specify the minimum y-axis value, maximum y-axis value, or range by including both.
-      - **Min**: Minimum y-axis value.
-      - **Max**: Maximum y-axis value.
+  - **auto** or **X-Axis**: Show all points with the same x value along the y-axis.
+  - **Y-Axis**: Show all points with the same y value along the x-axis.
+  - **X-Y Axis**: Show only the point being currently hovered over.
+
+###### X-Axis
+
+- **Generate X-Axis Tick Marks**: Select the method to generate x-axis tick marks:
+  - **Auto**: Select to automatically generate tick marks.
+  - **Custom**: To customize the number of x-axis tick marks, select this option, and then enter the following:
+    - **Total Tick Marks**: Enter the total number of timestamp ticks to display.
+    - **Start Tick Marks At**: Enter the time, in RFC3339 format, to start displaying ticks. Use the **Date Picker** field to automatically generate an RFC3339 formatted timestamp for this field.
+    - **Tick Mark Interval**: Enter the number of milliseconds in between each timestamp tick.
+
+###### Y-Axis
+
+- **Y Axis Label**: Enter the label for the y-axis.
+- **Y-Value Unit Prefix**: Select the prefix to add to the y-value:
+  - **None**: Select to add no prefix.
+  - **SI**: Select to add an International System of Units (SI) or metric prefix.
+  - **Binary**: Select to add a binary multiple prefix.
+- **Y Axis Prefix**: Enter the prefix to add to the y-value.
+- **Y Axis Suffix**: Enter the suffix to add to the y-value.
+- **Generate Y-Axis Tick Marks**: Select the method to generate y-axis tick marks:
+  - **Auto**: Select to automatically generate tick marks.
+  - **Custom**: To customize the number of y-axis tick marks, select this option, and then enter  the following:
+    - **Total Tick Marks**: Enter the total number of timestamp ticks to display.
+    - **Start Tick Marks At**: Enter the time, in RFC3339 format, to start displaying ticks.
```

Comment:
Curious why we added a date picker for the x-axis and not the y-axis

Reply:
Y-axis doesn't normally have date. time series data!

Reply:
Ah, so is the **Start Tick Marks At** field still relevant for the y-axis? The way I read the date picker for the x-axis was that it helps to automatically generate an RFC3339 formatted timestamp. If this field is valid, curious what happens if folks enter a non-formatted timestamp?

Reply:
For the y-axis could still be relevant. Here's an example of it with it set to `.5`
![image](https://user-images.githubusercontent.com/32681364/108554865-8e74f080-72a9-11eb-9f1a-070fa910fdbf.png)

Non-formatted timestamp:
![image](https://user-images.githubusercontent.com/32681364/108552820-aeef7b80-72a6-11eb-9b78-057a90f26102.png)